lib/models/database_tables.js

```diff
@@ -59,3 +64,9 @@ function shortHashKey(target) {
 function filterTablesWithUpdatedAt(table) {
     return table.hasOwnProperty('updated_at') && table.updated_at !== undefined && table.updated_at !== null;
 }
+
+const analysisTableRegex = /^analysis_[a-z0-9]{10}_[a-z0-9]{40}$/;
```

Comment:
This doesn't match `_CDB_IsAnalysisTableName`. Which one is correct?

Reply:
Aside from special regexp control chars, it should be `[a-f]` rather than `[a-z]`.

Reply:
I got it from https://github.com/CartoDB/Windshaft-cartodb/blob/master/lib/cartodb/api/map/analyses-catalog-controller.js#L140.

Changing it.